### ISSUE TITLE
The wallet goes into a locked state

### ISSUE THREAD

**Issue by SemenMartynov** [dashpay/dash#2104]
I've been working on Dash fork and noticed a behavior that's confusing me:
I have an unencrypted wallet and a few masternod. When I click on a "start missing" button (on a masternode tab) the wallet goes into a locked state and requires entering a passphrase, but this phrase has not been installed!

There is a temporary solution in the form of restarting the wallet.

**Comment by UdjinM6** [comment]
Thanks for reporting the issue however I can't reproduce it... What version/commit are you running and what OS are you having this issue at?

**Comment by SemenMartynov** [comment]
It was really hard to reproduce))))))))))
I work with a fork, and it's already gone pretty far... Frankly, I do not know the moment when the fork was made, but I compare all changes in the file masternodelist.cpp and see that it remained the same.
This issue does not depend on the OS; I tested on win10x64 and Ubuntu 16.04.
To play the bug I pressed several times  "start alias " and  "start missing". Now it has led me to the idea that somewhere arise race conditions... In either case, deleting this line solves the problem.
